torch/csrc/autograd/variable.h

```diff
@@ -90,7 +90,7 @@ struct TORCH_API Variable : public at::Tensor {
   //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
   // NOTE: These factory functions have to be friends to access the
-  // `Variable::Impl`. As a side effect, it allows us to keep them in the class.
+  // Variable's `impl_`. As a side effect, it allows us to keep them in the class.
```

Comment:
Is there really any reason to friend these now? We already have public methods for accessing impl on Tensor